frontend/src/utils/use-router-tabs-synchronization.ts

```diff
@@ -0,0 +1,35 @@
+import {onMounted, Ref, watch} from 'vue';
+import {useRoute, useRouter} from 'vue-router/composables';
+
+
+export default function useRouterTabsSynchronization(
```

Comment:
I'm not sure I understand the goal of this utility function compared to extracting a parent route and declaring shared parameters on it

Reply:
Let me refacto this using router links, I will also use pinia since I need to create specific component for each route